src/bindings/python/src/pyopenvino/pyopenvino.cpp

```diff
@@ -92,7 +92,7 @@ PYBIND11_MODULE(pyopenvino, m) {
     m.def(
         "set_batch",
         [](const std::shared_ptr<ov::Model>& model, int64_t value) {
-            return ov::set_batch(model, ov::Dimension(value));
+            ov::set_batch(model, ov::Dimension(value));
```

Comment:
Should we cover this here or make separate PR?

Reply:
it isn't connected with this issue but this change is too small and it doesn't affect anything so I think it would be better to leave it here.